cabal-ghc921.project

```diff
@@ -0,0 +1,139 @@
+packages:
```

Comment:
This file is mostly temporary and incomplete

Reply:
I ve updated it in #2131